libs/ledger-live-common/src/platform/types.ts

```diff
@@ -102,7 +102,10 @@ export type LiveAppManifest = {
   categories: string[];
   currencies: string[] | "*";
   visibility: Visibility;
+  highlight?: boolean;
   content: {
+    cta?: TranslatableString;
+    subtitle?: TranslatableString;
```

Comment:
We'll want to also update the `manifest-api` types too

Reply:
Indeed, here is the PR https://github.com/LedgerHQ/manifest-api/pull/332/